doc/sphinx/source/users-guide/base/simpleConsoleOutput.rst

```diff
@@ -0,0 +1,44 @@
+:title: Users Guide: Simple Console Output: Hello world
+
+Simple Console Output: Hello world
+==================================
+
+A good feature to learn first in any language is how to print things
+out to the console.  Although Chapel has a sophisticated set of I/O
+features, let's just focus on a simple capability to get started.
+
+
+The ``writeln()`` routine prints its list of comma-separated
```

Comment:
Should we refer to `writeln()` as a `proc` rather than a `routine` to be more consistent and pedagogical? Or alternatively, just note that routines/functions are call procs in Chapel.